AI_srvc/socketio_client.py

```
@@ -31,6 +31,12 @@
 
 log = logging.getLogger("socketio_client")
 
+HUMAN_ONLY_MODE = os.getenv("HUMAN_ONLY_MODE", "false").lower() == "true"
+if HUMAN_ONLY_MODE:
+    log.info("HUMAN_ONLY_MODE=true – AI service will not auto-connect to lobby slots.")
```

Ruff (0.15.9)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)